### PR TITLE
Initial static UI implementation of the polio dashboard

### PR DESCRIPTION
**Pull request by geoffreydhuyvetters**
### Added

- dashboard page (table)
- create modal
- round 1 and round 2 form

### CODE REVIEW

File: plugins/polio/js/src/components/Dashboard.js [hunk 1]
```diff
@@ -43,20 +50,20 @@ const useStyles = makeStyles(theme => ({
...
     },
     round1FormCalculations: {
-        marginTop: 64,
-        marginBottom: 32,
+        marginTop: theme.spacing(4),
+        marginBottom: theme.spacing(4),
     },
     input: {
-        marginTop: 32,
+        marginBottom: theme.spacing(2),
```
Comment (geoffreydhuyvetters) [inline plugins/polio/js/src/components/Dashboard.js:66]: Good catch 👍

**Review by xavierbarbosa — APPROVED** [review]
Looks good to me, I would already merge this so it does not continue to get bigger and harder to review PR. But will let @geoffreydhuyvetters take a look before merging.

File: plugins/polio/js/src/components/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,801 @@
...
+            </DialogContent>
+            <DialogActions className={classes.action}>
+                <Button onClick={onCancel} color="primary">
+                    Cancel
+                </Button>
+                <Button
+                    onClick={formik.handleSubmit}
+                    color="primary"
+                    autoFocus
+                    disabled={!formik.isValid}
```
Comment (xavierbarbosa) [inline plugins/polio/js/src/components/Dashboard.js:661]: Should we add `variant="contained"` so the button style would match other forms and looks more like a primary button?

**Review by kaiobrito — APPROVED** [review]

File: plugins/polio/js/src/components/Dashboard.js (hunk 2 above)
Comment (kaiobrito) [reply plugins/polio/js/src/components/Dashboard.js:656]: Done!

**Pull request merged**